docsk8s/administering/ls-k8s-stack-monitoring-external.asciidoc

```diff
@@ -0,0 +1,21 @@
+[[ls-k8s-monitor-external]]
+==== Ship metrics to external {es} cluster
+
+TIP: Be sure that you have CustomResourceDefinitions (CRDs) installed so that you can follow the example. Check out <<qs-set-up>> for set up info.
```

Comment:
```suggestion
TIP: Be sure that you have the Elastic CustomResourceDefinitions (CRDs) installed so that you can follow the example. Check out <<qs-set-up>> for set up info.
```